src/gui/plugins/view_angle/ViewAngle.hh

```diff
@@ -36,6 +36,13 @@ namespace gazebo
   {
     Q_OBJECT
 
+    /// \brief gui camera pose (QList order is x, y, z, roll, pitch, yaw)
+    Q_PROPERTY(
+      QList<QString> camPose
```

Comment:
Could this store `double`s instead of `QString`s?

Reply:
0cdb38b3fdf9d3bebd4c2cc4a3d362103453de07